src/effects/useTokenBalances.ts

```diff
@@ -0,0 +1,48 @@
+import { useState, useEffect } from 'react'
+import { Network, TokenBalanceDetails } from 'types'
+import { depositApi } from 'api'
+import { tokenApi } from 'api'
+
+interface UseTokenBalanceResult {
+  balances: TokenBalanceDetails[] | undefined
+  error: boolean
+}
+
+async function _getBalances(): Promise<TokenBalanceDetails[]> {
+  const tokens = tokenApi.getTokens(Network.Rinkeby)
+  const userAddress = '0x1111111111111111111111111111111111111111'
```

Comment:
Just to be clear, this is a temporary value, we'll later get this value as input from the connected wallet, correct?

Reply:
You'll see it in one of the standing PRs :)